UnitedIncome/components
# [APP-4809] Use vertical flex layout for InfoBox

### Description
Design now wants these cards to be able to grow vertically to fit their containers if need be. This requires us to make the content portion of the card flexible.

### Testing Instructions
1. Go to the `InfoBox` page in Storybook
2. You can use the `height` field to adjust the container height; the card should adjust with it

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4809)
